packages/codebytes/src/helpers/index.ts

```diff
@@ -27,7 +29,16 @@ export const getOptions = (): CodebyteOptions => {
   };
 };
 
-export const trackClick = (target: string) => {
+export const trackClick = (
+  target: string,
+  trackingData: Omit<UserClickData, 'target'>
+) => {
+  if (trackingData) {
```

Comment:
maybe we make comment about tech debt ticket here/mention homepage?

Reply:
100% good call. just wanted to see if you and @BandanaKM were okay with this stopgap approach

Reply:
yeah, think this works as a quick way to pass in the correct tracking data from monolith to avoid getting the weird values for `clientName` in all of our `userClick` events in the monolith.

we can definitely refactor this as a fast follow. ideally we can get rid of this file in the future and just have getOptions and all the relevant dependencies in the next.js version, since they were primarily intended for the query params

if we do end up passing in an object later with the tracking functions, we can definitely do that